.github/workflows/ci.yml

```diff
@@ -155,7 +177,7 @@ jobs:
       - name: "Set up Python"
         uses: "actions/setup-python@v2"
         with:
-          python-version: "3.10"
+          python-version: "3.9"
```

Comment:
Reason to go to an older Python version? Template?

Reply:
Yup, it's in the template. Should I bump it to 3.10 / 3.11 in the template? Newer Python versions has better performance.

Reply:
No need to bump. I think this is yet another good case for cookie version 1.2